### PR TITLE
feat: add examples book

### PR DESCRIPTION
**Pull request by debs-obrien**

### CODE REVIEW

**Review by Kronhyx — APPROVED** [review]

**Comment by debs-obrien** [comment]
@Atinux this is the examples book which with descriptions and button to open code sandbox within the other books. Live demo can be seen here: 
https://feat-codesandboxes-description--nuxt.netlify.app/guides/examples/hello-world

We can modify intro text or see example button if you prefer. 

Descriptions are in two places so we have to maintain them in both the examples book and in the guides books under the corresponding chapters

All feedback welcome @pi0 @manniL @danielroe

**Review by Atinux — DISMISSED** [review]
Amazing work!

File: content/en/guides/directory-structure/nuxt-config.md [hunk 1]
```diff
@@ -327,6 +327,23 @@ See more on the [transition property](/guides/configuration-glossary/configura
 
 </base-alert>
 
+### Example
+
+<example-intro></example-intro>
+
+`nuxt.config.js` contains :
+
+- the `env` property to add the URL of our API.
```
Comment (Atinux) [inline content/en/guides/directory-structure/nuxt-config.md:336]: Could we use the `publicRuntimeConfig` instead of `env` for this example?

Comment (debs-obrien) [reply content/en/guides/directory-structure/nuxt-config.md:336]: you mean also change it in the docs: we only document env property and not publicRuntimeConfig / privateRuntimeConfig

https://nuxtjs.org/guides/directory-structure/nuxt-config#env

do we remove env from docs altogether or just add the extra ones to this page and use in example? @Atinux

Comment (debs-obrien) [reply content/en/guides/directory-structure/nuxt-config.md]: done

**Comment by debs-obrien** [comment]
runtimeConfig added to docs as well as example: https://github.com/nuxt/nuxtjs.org/pull/890

**Comment by Atinux** [comment]
I think we should use publicRuntimeConfig.axios.baseURL, see https://axios.nuxtjs.org/options

File: + [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Merge branch 'master' into feat-codesandboxes-description
```
Comment (pi0) [inline +:1]: Accidental add?

File: i18n/fa-IR.js [hunk 3]
```diff
@@ -291,6 +291,9 @@ module.exports = {
     msg1: 'مشارکت در این صفحه بسته شده است. اگر می‌‌خواهید مشارکت کنید',
     link: 'مستندات جدید'
   },
+  example: {
+    intro: 'In this example:'
```
Comment (pi0) [inline i18n/fa-IR.js:295]:
```suggestion
    intro: 'در این مثال:'
```

File: + (hunk 2 above)
Comment (debs-obrien) [reply +:1]: why? there were some modifications to the docs so I needed to fix merge issues locally so merged master into the branch.

Comment (pi0) [reply +:1]: Actually it is a file named `+`  and staged 😁 

https://github.com/nuxt/nuxtjs.org/blob/a8b4d84b3c01731a6da064e731468da1616a6b15/%2B

Comment (debs-obrien) [reply +]: wow. how did that get there. very weird. Thanks. Removed

File: i18n/fa-IR.js (hunk 3 above)
Comment (febrihidayan) [reply i18n/fa-IR.js]: @pi0 I'm not sure it's written that way. should be like this `:در این مثال`

**Pull request merged**